awpy/analytics/wpa.py

```diff
@@ -30,9 +31,57 @@ def round_win_probability(ct_score: int, t_score: int, map_name: str) -> dict:
     Args:
         ct_score (int): CT Score
         t_score (int): T Score
-        map_name (str): Map the demo is from
+        map_name (str): Map name the demo is from
 
     Returns:
-        A dictionary containing the CT game win, T game win and Draw probabilities
+        A dictionary containing the game win probability
     """
-    raise NotImplementedError
+    # Load the data from the json file
+    wpa_data = json.loads("../data/wpa/wpa.json")
+
+    # Get the map id from the map name
+    map_id = _get_mapid(map_name)
+
+    # Determine the team based on the scores
+    team = "CT" if ct_score > t_score else "TERRORIST"
+
+    # If the map_id, team, ct_score, or t_score is not in the data, return
+    if (
+        str(map_id) not in wpa_data
+        or team not in wpa_data[str(map_id)]
+        or str(ct_score) not in wpa_data[str(map_id)][team]
+        or str(t_score) not in wpa_data[str(map_id)][team][str(ct_score)]
+    ):
+        return {"error": "Invalid input"}
+
+    # Get the win probability
+    win_prob = wpa_data[str(map_id)][team][str(ct_score)][str(t_score)]
+
+    return {
+        "win_probability": win_prob,
+    }
+
```

Comment:
- I do not quite get the thing about determining the team from the score?
- Shouldnt the CT win probability just be data[CT][ct_score][t_score] and the T probability be data[T][ct_score][t_score]
- I think we can just return a dict with the ct, t and draw probability.

We really do not need to return a dict with a singular item.

Reply:
I was trying to do some further calcs but figured we don't need all that. Will make it much simpler like:

`# Get the win probability from the loaded data
    ct_win_probability = wpa_data[str(map_id)]["CT"][str(ct_score)][str(t_score)]
    t_win_probability = wpa_data[str(map_id)]["T"][str(ct_score)][str(t_score)]

    return {"CT": ct_win_probability, "T": t_win_probability}`